vllm/engine/arg_utils.py

```diff
@@ -1804,3 +1814,20 @@ def human_readable_int(value):
 
     # Regular plain number.
     return int(value)
+
+
+def get_model_path(model: Union[str, Path], revision: Optional[str] = None):
+    if os.path.exists(model):
+        return model
+
+    common_kwargs = {
+        "local_files_only": huggingface_hub.constants.HF_HUB_OFFLINE,
+        "revision": revision,
+    }
```

Comment:
```suggestion
    assert huggingface_hub.constants.HF_HUB_OFFLINE
    common_kwargs = {
        "local_files_only": huggingface_hub.constants.HF_HUB_OFFLINE,
        "revision": revision,
    }
```
Let's add a assertion here to avoid unexpected network connection.